frontend/kesaseteli/youth/src/types/youth-form-data.d.ts

```diff
@@ -0,0 +1,10 @@
+import School from 'kesaseteli/youth/types/School';
+import YouthApplication from 'kesaseteli/youth/types/youth-application';
+
+type YouthFormData = Omit<YouthApplication, 'school'> & {
+  school?: School | null;
+  unlisted_school?: string | null;
+  termsAndConditions: boolean;
```

Comment:
You're right that it does not make anymore sense. Fields that are saved to the backend have snake case and form dependent has camelCase. Now `unlisted_school` isnt going to backend so it could be camelCase. There will be some changes in next pr, so I will fix this there